main.js

```diff
@@ -49,41 +50,39 @@ var textInput = document.querySelector('.message-input');
 var textInputLabel = document.querySelector('.input-error');
 var errorMessage = document.querySelector('.receive-error');
 var clearMessageButton = document.querySelector('.clear-message');
-var currentMessage = '';
 
 //--------------------/event listeners/----------------------//
-// window.addEventListener('load', )
+
 receiveMessageButton.addEventListener('click', displayMessage);
 addMessageButton.addEventListener('click', displayForm);
 submitButton.addEventListener('click', submitMessage);
+clearMessageButton.addEventListener('click', showBellImage);
 addMantraSelect.addEventListener('click', hideErrorMessage);
 addAffirmationSelect.addEventListener('click', hideErrorMessage);
 mantraSelect.addEventListener('click', hideErrorMessage);
 affirmationSelect.addEventListener('click', hideErrorMessage);
-clearMessageButton.addEventListener('click', showBellImage);
 
 //--------------------/functions/----------------------//
-//transition to the data model!
-//create helper function that checks to see if newAff/newMant string exists in the aff or mant arrays.
-//create logic statement to pass it to the correct array accordingly.
 
 function submitMessage(e) {
   e.preventDefault();
   hideForm();
-  if (!addMantraSelect.checked && !addAffirmationSelect.checked) {
+  if (!addMantraSelect.checked && !addAffirmationSelect.checked || !textInput.value) {
     displayForm();
     textInputLabel.classList.toggle('hidden');
 
   } else if (addAffirmationSelect.checked) {
     textInputLabel.classList.add('hidden');
-    viewMessage.innerText = `${textInput.value}`;
-    affirmations.push(`${textInput.value}`);
+    currentMessage = `${textInput.value}`;
+    viewMessage.innerText = currentMessage;
+    updateAffirmations();
     hideErrorMessage();
 
   } else if (addMantraSelect.checked) {
     textInputLabel.classList.add('hidden');
-    viewMessage.innerText = `${textInput.value}`;
-    mantras.push(`${textInput.value}`);
+    currentMessage = `${textInput.value}`;
+    viewMessage.innerText = currentMessage;
+    updateMantras();
     hideErrorMessage();
   }
 }
```

Comment:
A lot of repetition in displayMessage(), it looks like you always want to hideImage and hideForm regardless of what else happens, so maybe you can reduce the amount of logic in this function